dao/authentication_dao.go

```diff
@@ -255,7 +255,7 @@ func (a *AuthenticationDaoImpl) Create(auth *m.Authentication) error {
 
 	number, ok := out.Data["version"].(json.Number)
 	if !ok {
-		return nil
+		return fmt.Errorf("failed to cast vault version number to string")
```

Comment:
Small nitpick — would suggest using `errors.New` here, since you're not using any verbs in the string

Reply:
alrighty, i usually just do fmt since its already imported. but all good haha updated.

Reply:
That's actually a good point